app/src/main/java/com/example/android/weatherapp/WeatherRepository.kt

```diff
@@ -0,0 +1,91 @@
+package com.example.android.weatherapp
+
+import com.example.android.weatherapp.models.*
+import com.example.android.weatherapp.utilities.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import retrofit2.HttpException
+
+class WeatherRepository {
+
+    private val weatherForecast: ForecastData = ForecastData()
+    private val helper: Helper = Helper()
```

Comment:
Rename Helper to some more descriptive name like WeatherHelper maybe?

Reply:
Renamed it to **formatHelper**